src/PSRule.Rules.Azure/rules/Azure.AKS.Rule.ps1

```diff
@@ -305,6 +305,21 @@ Rule 'Azure.AKS.EphemeralOSDisk' -Ref 'AZR-000287' -Level Warning -Type 'Microso
     }
 }
 
+# Synopsis: Use kube-audit-admin instead of kube-audit to capture administrative actions in AKS clusters.
+Rule 'Azure.AKS.AuditAdmin' -Ref 'AZR-000445' -Type 'Microsoft.ContainerService/managedClusters' -Tag @{ release = 'GA'; ruleSet = '2024_09'; 'Azure.WAF/pillar' = 'Cost Optimization'; } {
+    $kubeAuditLogs = @(GetSubResources -ResourceType 'Microsoft.Insights/diagnosticSettings' |
+        Where-Object { $_.properties.logs | Where-Object { $_.category -eq 'kube-audit' -and $_.enabled } } )
+
+    if ($kubeAuditLogs.Count -eq 0) {
+        return $Assert.Pass()
+    }
+
+    foreach ($kubeAuditLog in $kubeAuditLogs) {
+        $Assert.HasDefaultValue($kubeAuditLog, "properties.logs[?@category == 'kube-audit'].category", 'kube-audit-admin').
+        Reason($LocalizedData.AKSAuditAdmin, $kubeAuditLog.name)
+    }
+}
+  
```

Comment:
You are correct, @BernieWhite. This approach is better. I over-engineered the solution, and now I see it can be resolved with less complexity. Thanks.